controllers/keda/scaledobject_controller.go

```diff
@@ -580,10 +583,18 @@ func (r *ScaledObjectReconciler) updatePromMetrics(scaledObject *kedav1alpha1.Sc
 	}
 	metricsData.triggerTypes = triggerTypes
 
+	isPausedObject := scaledObject.HasPausedAnnotation()
+	active := false
+	if isPausedObject {
+		active = true
+	}
+
+	metricscollector.RecordScalerPaused(namespacedName, scaledObject.Name, active)
```

Comment:
better to move this to `reconcileScaledObject()` where we already check paused annotations